src/System.Text.Json/tests/Serialization/SampleTestData.Order.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization.Tests.Schemas.Order
+{ 
+    public class Order
```

Comment:
Nit: it's generally frowned upon having a class be in a namespace of the same name

Reply:
will update the namespace